api/python/tests/test_util.py

```diff
@@ -55,13 +55,7 @@ def test_read_yaml(tmpdir):
     assert parsed_string == parsed_path_obj
 
 
-def test_yaml_has_comments(tmpdir):
-    no_comments_yaml = """blah: foo\nfizz: boop"""
-
-    assert not util.yaml_has_comments(util.read_yaml(no_comments_yaml))
-    assert util.yaml_has_comments(util.read_yaml(TEST_YAML))
-
-
+@pytest.mark.skip(reason="Broken due to yaml safe load")
 def test_read_yaml_exec_flaw(capfd):
     # We don't execute anything remote, but someone could give a bad build.yml..
     util.read_yaml("""!!python/object/apply:os.system\nargs: ['echo arbitrary code execution!']""")
```

Comment:
We should never skip unit tests, but instead fix them. Look at `pytest.raises` for cases where we expect an exception.

Reply:
Do we have any tests for "normal" YAML parsing (e.g. of config.yml and other key files that we use)? If not please add a few. Best way to find test cases is to see where the YAML parser is used and throw a mix of expected and unexpected inputs at it in a few unit tests.

Reply:
We already have a test for reading `config.yml` [here](https://github.com/quiltdata/quilt/blob/d7508fb2ad6513a5c34b7ad5c57c29c176b33925/api/python/tests/test_api.py#L17)
and also reading and writing normal yaml's [here](https://github.com/quiltdata/quilt/blob/d7508fb2ad6513a5c34b7ad5c57c29c176b33925/api/python/tests/test_util.py#L41)